README.md

```diff
@@ -54,12 +54,23 @@ This is the method to use if you do not have an
 All you need to do is upload your configuration file and add its URL to
 your import map as a module named **config-file**. If you are serving your
 microfrontends from your OpenMRS server, you can simply add your config
-file to your server's `frontend/` directory. Your import map will then look like
+`config.js/` file to your server's `frontend/` directory. Your import map will then look like
+
+```config.js
+  exports = {
+  default: {
+    "@openmrs/esm-login-app": {
+      "chooseLocation": {
+        "enabled": true
+      }
+    }
+  }
+}
 
-```json
+```import json
```

Comment:
Why import in front? (I actually don't know what that does)

Reply:
specifying the file name.

Reply:
Ah okay - but it is not shown (rendered) by GitHub, right? So only interesting for the MD reader, right?

Reply:
its upto date now